include/interface/blas1_interface.h

```diff
@@ -76,6 +76,33 @@ template <typename executor_t, typename container_0_t, typename container_1_t,
 typename executor_t::policy_t::event_t _dot(
     executor_t &ex, index_t _N, container_0_t _vx, increment_t _incx,
     container_1_t _vy, increment_t _incy, container_2_t _rs);
+
+/**
+ * \brief Computes the inner product of two vectors with extended precision
+ * accumulation and adds a scalar to the result (Asynchronous version that
+ * returns an event)
+ * @tparam executor_t Executor type
+ * @tparam container_0_t Buffer Iterator
+ * @tparam container_1_t Buffer Iterator
+ * @tparam container_2_t Buffer Iterator
+ * @tparam index_t Index type
+ * @tparam increment_t Increment type
+ * @param ex Executor
+ * @param _N Input buffer sizes. If size 0, the result will be sb.
+ * @param sb scalar to add to the results of the inner product.
+ * @param _vx Buffer holding input vector x
+ * @param _incx Increment in x axis
```

Comment:
I think it is better to describe this as a stride. There are no x and y axes.

Reply:
Same as above comment. I copied this from _dot() description. Happy to update since it's more clear to refer to them as strides